.gitignore

```diff
@@ -51,6 +51,9 @@ coverage.xml
 .pytest_cache/
 cover/
 
+# pytest output files
```

Comment:
Those are kafe2 files created during the tests to check the write to and load from file features including kafe2go.

Reply:
I changed this comment, hopefully the description is now fine